mod/node-api/backend/backend.go

```
@@ -42,8 +42,8 @@ type Backend[
 		ValidatorT, ValidatorsT, WithdrawalT,
 	],
 	BeaconStateMarshallableT any,
-	BlobSidecarsT any,
-	BlockStoreT BlockStore[BeaconBlockT],
+	BlobSidecarT BlobSidecar[BeaconBlockHeaderT],
+	BlobSidecarsT BlobSidecars[BlobSidecarsT, BlobSidecarT], BlockStoreT BlockStore[BeaconBlockT],
```

golangci-lint

the line is 95 characters long, which exceeds the maximum of 80 characters.

(lll)